common/mariadb/templates/initdb/_init.sql.tpl

```diff
@@ -1,3 +1,25 @@
-CREATE DATABASE IF NOT EXISTS {{.Values.name}} CHARACTER SET utf8 COLLATE utf8_general_ci;
-GRANT ALL PRIVILEGES ON {{.Values.name}}.* TO {{.Values.global.dbUser}}@localhost IDENTIFIED BY '{{include "db_password" .}}';
-GRANT ALL PRIVILEGES ON {{.Values.name}}.* TO {{.Values.global.dbUser}}@'%' IDENTIFIED BY '{{include "db_password" .}}';
+{{- if not .Values.databases }}
+CREATE DATABASE IF NOT EXISTS {{ .Values.name }};
+{{- else }}
+    {{- range .Values.databases }}
+CREATE DATABASE IF NOT EXISTS {{ . }};
```

Comment:
Same as above. `CHARACTER SET utf8 COLLATE utf8_general_ci` should be added to the create DB statement
```suggestion
CREATE DATABASE IF NOT EXISTS {{ . }} CHARACTER SET utf8 COLLATE utf8_general_ci;
```

Reply:
See [before](https://github.com/sapcc/helm-charts/pull/3555/files#r889121676)